lib/lunchbot/lunchbot_data/options.ex

```diff
@@ -3,11 +3,11 @@ defmodule Lunchbot.LunchbotData.Options do
   import Ecto.Changeset
 
   schema "options" do
+    belongs_to :option_heading, Lunchbot.LunchbotData.OptionHeading
     field :extra_price, :integer
     field :extras, :boolean, default: false
     field :is_required, :boolean, default: false
     field :name, :string
-    field :option_heading_id, :integer
```

Comment:
why did you get rid of the option_heading_id?

Reply:
```.ex
belongs_to :option_heading, Lunchbot.LunchbotData.OptionHeading
```
is going to default to becoming a field named `option_heading_id`, so it replaces `option_heading_id`